### PR TITLE
Usertoken

### PR DESCRIPTION
**Pull request by fcavazzoli**
Now it is all working according to the task in trello :)

### CODE REVIEW

**Review by jhonmunoz — DISMISSED** [review]

**Review by cquiro — CHANGES_REQUESTED** [review]
Add the log folder to the .gitignore file.

File: .rubocop.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@ Documentation:
    Enabled: false
 LineLength:
    Max: 99
+Rails/ApplicationRecord:
+  Enabled: false
```
Comment (floresv) [inline .rubocop.yml]: Why? 🤔

File: Gemfile [hunk 2]
```diff
@@ -40,7 +40,10 @@ gem 'sass'
 gem 'scss-lint'
 
 #gemas para token
+gem 'haml'
```
Comment (floresv) [inline Gemfile]: Why?

File: Gemfile [hunk 3]
```diff
@@ -40,7 +40,10 @@ gem 'sass'
 gem 'scss-lint'
 
 #gemas para token
+gem 'haml'  
+gem 'rack-cors', :require => 'rack/cors'
```
Comment (floresv) [inline Gemfile]: Are you really need this? https://github.com/lynndylanhurley/devise_token_auth/blob/master/docs/config/cors.md

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
+  extend Devise::Models
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```
Comment (floresv) [inline app/models/user.rb]: Remove commented code

**Review by floresv — CHANGES_REQUESTED** [review]
I agree with 🚗 🔑

**Review by floresv — COMMENTED** [review]
It's good but remove `log/development.log`

**Comment by floresv** [comment]
I still see the log development file, please remove it.

**Review by floresv — APPROVED** [review]
Looks good to me

**Review by cquiro — APPROVED** [review]

**Pull request merged**